sdk/modelsrepository/Azure.Iot.ModelsRepository/src/DependencyResolutionOption.cs

```diff
@@ -4,22 +4,23 @@
 namespace Azure.Iot.ModelsRepository
 {
     /// <summary>
-    /// The dependency processing options.
+    /// The model dependency resolution options.
     /// </summary>
     public enum DependencyResolutionOption
     {
         /// <summary>
-        /// Do not process external dependencies.
+        /// Disable model dependency resolution.
         /// </summary>
         Disabled,
 
         /// <summary>
-        /// Enable external dependencies.
+        /// Enable model dependency resolution.
         /// </summary>
         Enabled,
 
         /// <summary>
-        /// Try to get external dependencies using .expanded.json.
+        /// Try to get pre-computed model dependencies using .expanded.json.
+        /// If the model expanded form does not exist fall back to normal dependency processing.
         /// </summary>
         TryFromExpanded
```

Comment:
if this is the default option (which I think it is) let's move it to the top of the enum declaration.

Reply:
Done.